apps/ern-reconnet/src/views/documents-page.vue

```diff
@@ -7,14 +7,12 @@
     />
     <PageSection aria-labelledby="welcome-title">
       <h2 id="welcome-title">ERN ReCONNET registry Documents</h2>
-      <MessageBox>
-        <p>This page is under construction.</p>
-      </MessageBox>
+      <FileList table="Files" labelsColumn="name" fileColumn="file" />
```

Comment:
The file list component appears to sit directly under the title. I think some styles are missing. I'll have a look at it further.

Reply:
Added in #a1158ba